Remove mentions of the FPS chart in the Performance docs (#2242)
MicrosoftDocs/edge-developer: microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md

> The main metric for measuring the performance of any animation is frames per second (FPS).  Users are happy when animations run at 60 FPS.

Reviewer: I think this should be cut and the `### Analyze frames per second` section title. We don't have FPS tracked any more in the Performance tool so everything related to FPS needs to be removed

Author: done

1. Look at the **FPS** chart, shown below.  Whenever a red bar is displayed above **FPS**, it means that the framerate dropped so low that it's probably harming the user experience.  In general, the higher the green bar, the higher the FPS.

>    ![The FPS chart.](../media/evaluate-performance-performance-fps-chart.msft.png)

Reviewer 2: Before we merge this, can you check if this image is used anywhere else in the docs? If not, then please delete the image file. Thanks!

1. Hover over the **CPU** or **NET** charts.  DevTools shows a screenshot of the page at that point in time.  Move your mouse left and right to replay the recording.  The action is called _scrubbing_, and it's useful for manually analyzing the progression of the performance recording.

   ![View a screenshot of the page around the 2500ms mark of the recording.](../media/evaluate-performance-performance-screenshot-hover.msft.png)

> 

Reviewer: I think the next step also needs to be removed. The **Frames** section no longer displays FPS for each particular frame. I can't reproduce the screenshot shown here.

> 1. Look at the **FPS** chart, shown below.  Whenever a red bar is displayed above **FPS**, it means that the framerate dropped so low that it's probably harming the user experience.  In general, the higher the green bar, the higher the FPS.
> 

Reviewer 3: ```suggestion
```

>    ![The FPS chart.](../media/evaluate-performance-performance-fps-chart.msft.png)

Reviewer 2: Can you explain why you removed this image? The text before says "look at the FPS chart below", but there isn't a chart below anymore. So, it's a bit confusing. Shouldn't we just keep this image?

Author: From Reeva:  [10:06 AM] Reeza Ali
[9:47 AM] Reeza Ali
Jason Andrew (Wipro Designit Services, Inc.)If that's the case the work item needs to be revise and everything redone. I was just worker bee doing it, which is how it came up, but I can revise. Reeza Ali? Shall I do that?
Thanks for asking for clarification. 
 
I met with Zoher (1 month ago), and then Zoher and Rachel about 1-2 weeks ago. This was the guidance and changes they recommended we make to the doc. 
 
In terms of next steps: Let's button it up and send it to them for review. They can then guide us on what might need to change or not. 
 
[9:48 AM] Reeza Ali
Also, Rachel is planning to overhaul the performance docs in Dec.

> 1. The **CPU** chart is displayed.  The colors in the **CPU** chart correspond to the colors in the **Summary** panel, at the bottom of the **Performance** tool.  The fact that the **CPU** chart is full of color means that the CPU was maxed out during the recording.  Whenever the CPU is maxed out for long periods, that's an indicator that you should find ways to do less work.

Reviewer 3: ```suggestion
1. The **CPU** chart is displayed along the top.  The colors in the **CPU** chart correspond to the colors in the **Summary** panel, at the bottom of the **Performance** tool.  The **CPU** chart shows that these regions make up a large area, meaning that the CPU was maxed out during the recording.  Whenever the CPU is maxed out for long periods, that's an indicator that you should find ways to do less work.
```

   ![The CPU chart and Summary panel.](../media/evaluate-performance-performance-cpu-chart.msft.png)

> 1. Hover over the **CPU**, or **NET** charts.  DevTools shows a screenshot of the page at that point in time.  Move your mouse left and right to replay the recording.  The action is called _scrubbing_, and it's useful for manually analyzing the progression of animations.

Reviewer 2: Why are we removing FPS from the first sentence? Does hovering over the FPS chart not do anything?

Author: I don't know the reason why. It was the work item. Shall I ask the SMEs? https://microsoft.visualstudio.com/Edge/_queries/edit/41648598/?triage=true

Reviewer 3: FPS chart does not exist.

> 1. Hover over the **CPU**, or **NET** charts.  DevTools shows a screenshot of the page at that point in time.  Move your mouse left and right to replay the recording.  The action is called _scrubbing_, and it's useful for manually analyzing the progression of animations.

Reviewer 3: ```suggestion
1. Hover over the **CPU** or **NET** charts.  DevTools shows a screenshot of the page at that point in time.  Move your mouse left and right to replay the recording.  The action is called _scrubbing_, and it's useful for manually analyzing the progression of the performance recording.
```
